pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -114,12 +113,12 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *v1.Pod, node
 	processPod := func(existingPod *v1.Pod) error {
 		existingPodNode, err := ipa.info.GetNodeInfo(existingPod.Spec.NodeName)
 		if err != nil {
```

Comment:
I would just do this instead of the block below:

```suggestion
		if err != nil || existingPodNode == nil {
```